packages/nextra/src/client/components/search.tsx

```diff
@@ -53,14 +53,7 @@ const DEV_SEARCH_NOTICE = (
     </p>
     <ol className="_mt-2 _list-decimal">
       <li>Build your app with `next build`</li>
-      <li>
-        Copy `.next/static/chunks/pagefind` folder to a different location
-      </li>
-      <li>Start your app in dev mode using `next dev`</li>
-      <li>
-        Copy `pagefind` folder into `.next/static/chunks/<b>app</b>` or
-        `.next/static/chunks/<b>app/[lang]</b>` for i18n website
-      </li>
+      <li>Restart your app in dev mode using `next dev`</li>
```

Comment:
I removed unneeded steps 😅 feel free to polish/rephrase as you think should be better 😉

Reply:
Search isn't available in development because Nextra 4 uses the Pagefind package, which indexes built `.html` files instead of `.md`/`.mdx`. To test search during development, run `next build` and then restart your app with `next dev`.

I asked chat gpt to make shorter  😂

Reply:
It’s clear to me!